assets/js/scenes.js

```diff
@@ -25,6 +26,8 @@ const generateScenes = () => {
 
     // spawn the egg jar
     spawnEggJar(200, 200, WHITE);
+    spawnBabyYoda(500, 200, RED);
```

Comment:
We can remove the arguments here and just called it as `spawnBabyYoda()` since the position will always be the same and we now have sprites instead of colours.